src/js/components/Ballot/BallotTitleHeader.jsx

```diff
@@ -102,13 +105,58 @@ class BallotTitleHeader extends Component {
     }
   }
 
-  showSelectBallotModalEditAddress = () => {
+  showSelectBallotModalEditAddress = (buttonId) => {
+    console.log('Passed buttonId:', buttonId);
     const { linksOff } = this.props;
     // console.log('BallotTitleHeader showSelectBallotModalEditAddress linksOff:', linksOff);
     if (!linksOff) {
       const showEditAddress = true;
       const showSelectBallotModal = true;
       // this.props.toggleSelectBallotModal('', showEditAddress, false);
+      const { location: { pathname: currentPathname } } = window;
+      const page = lookupPageNameAndPageTypeDict(currentPathname);
+
+      const address = VoterStore.getTextForMapSearch();
```

Comment:
VoterStore.getTextForMapSearch() gets the previous value, and I believe we want the value after the change (which is a bit messy).  @DaleMcGrew needs to decide what he wants here.  I left a longer description in https://wevoteusa.atlassian.net/issues/WV-1096

Reply:
Thank you @SailingSteve for this dataLayer review. We are capturing the existing address at the start of the edit address process with this ticket. We will have another dataLayer ticket (to be created) when the voter saves a *new* address. @ilonagog